topics/uportal/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Apereo
+display_name: uPortal
+github_url: https://github.com/Jasig/uPortal
+logo: uportal.png
+related: apereo, portlet, soffit 
+released: December 1999
+short_description: uPortal is an enterprise portal for education institutions.
+topic: uportal
+url: https://www.apereo.org/projects/uportal
+wikipedia_url: https://en.wikipedia.org/wiki/UPortal
+---
+uPortal is an open source enterprise portal framework built by and for higher education institutions, K-12 schools and research communities.
```

Comment:
You could expand this description by another sentence or two, if you wanted.

Can you add a comma after `schools`?

Reply:
Thanks :bowing_man:
Updated with Oxford comma and some addition description.